### PR TITLE
feat: write sample for creating key for service account

### PR DESCRIPTION
**Pull request by BigBlackWolf**
## Description

Fixes #<ISSUE-NUMBER>

Note: Before submitting a pull request, please open an issue for discussion if you are not associated with Google.

## Checklist
- [ ] I have followed [Sample Guidelines from AUTHORING_GUIDE.MD](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md)
- [ ] README is updated to include [all relevant information](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md#readme-file)
- [ ] **Tests** pass:   `nox -s py-3.9` (see [Test Environment Setup](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md#test-environment-setup))
- [ ] **Lint** pass:   `nox -s lint` (see [Test Environment Setup](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md#test-environment-setup))
- [ ] These samples need a new **API enabled** in testing projects to pass (let us know which ones)
- [ ] These samples need a new/updated **env vars** in testing projects set to pass (let us know which ones)
- [ ] This sample adds a new sample directory, and I updated the [CODEOWNERS file](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/.github/CODEOWNERS) with the codeowners for this sample
- [ ] This sample adds a new **Product API**, and I updated the [Blunderbuss issue/PR auto-assigner](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/.github/blunderbuss.yml) with the codeowners for this sample
- [ ] Please **merge** this PR for me once it is approved

### CODE REVIEW

File: iam/cloud-client/snippets/create_key.py
Comment (m-strzelczyk) [inline iam/cloud-client/snippets/create_key.py:45]: Perhaps we should return `json_key_data` or even better, the whole `key`, as you warn that it cannot be obtain later - let's not lose it that fast :P

**Review by m-strzelczyk — APPROVED** [review]

File: iam/cloud-client/snippets/create_key.py
Comment (BigBlackWolf) [reply iam/cloud-client/snippets/create_key.py:45]: Yeah, makes sense 😅

**Comment by m-strzelczyk** [comment]
This fails, because it's dependant on #11444 will be fine to merge once #11444 is merged.

**Review by m-strzelczyk — APPROVED** [review]

**Pull request merged**
